DANS-KNAW/easy-convert-bag-to-deposit
# DD-456: Include AMD in convert-to-deposit

Fixes DD-456, DD-480

When applied it will
--------------------
* read `amd.xml` from Fedora when source=`VAULT`
* then write amd.xml into `metadata` and `data/easy-migration` folders

Where should the reviewer @DANS-KNAW/easy start?
------------------------------------------------

How should this be manually tested?
-----------------------------------
* run easy-convert-bag-to-deposit with parameter -s VAULT, with a bag where you don't have emd.xml and amd.xml
(amd.xml should appear in the resulting bag in metadata and data/easy-migration folders)
* run easy-convert-bag-to-deposit with parameter -s FEDORA, with a bag where you do have emd.xml and amd.xml
(emd.xml should appear in the resulting bag in data/easy-migration folder, but amd.xml not)


Related pull requests on github
-------------------------------

repo                       | PR                | note
-------------------------- | ----------------- | ----
easy-                      | [PR#](PRlink)     | ..
